inc/ThirdParty/Themes/Divi.php

```diff
@@ -172,4 +174,24 @@ public function disable_divi_jquery_body() {
 		}
 
 	}
+
+	/**
+	 * Excludes Divi's CSS files from CSS combination
+	 *
+	 * @since 3.10
+	 *
+	 * @param array $exclude_css An array of CSS  to be excluded.
+	 *
+	 * @return array the updated array of paths
+	 */
+	public function exclude_divi_css_from_combine( $exclude_css ) {
+
+		if ( (bool) $this->options->get( 'remove_unused_css', 0 ) ) {
+			return $exclude_css;
+		}
```

Comment:
We don't need to check for the `remove_unused_css` value. If RUCSS is enabled, there won't be any minify/combine happening anyway.

What we need to check for is if `minify_concatenate_css` is enabled, and bail-out if it isn't.